packages/dropdowns.legacy/src/styled/multiselect/StyledMultiselectInput.ts

```diff
@@ -13,22 +13,26 @@ const COMPONENT_ID = 'dropdowns.multiselect_input';
 
 interface IStyledMultiselectInputProps {
   isCompact: boolean;
```

Comment:
Should this be `$isCompact`?

Reply:
No in this case. This case is tricky to understand. Using `$isCompact` led to a snapshot mismatch.

TL&DR;

`StyledMultiselectInput` extends `StyledInput`, which in turns extends `Input`, a `React FC` that returns a `StyledTextInput`. `Input` only accepts `isCompact`, which is received by `StyledTextInput` after being renamed as transient prop `$isCompact`.